Taskfile.yml

```diff
@@ -61,6 +63,27 @@ tasks:
       - cmd: echo "opslevel-go workspace ready!"
         silent: true
 
+  has-latest-opslevel-go:
+    desc: Check if latest release of opslevel-go in go.mod
+    dir: "{{.SRC_DIR}}"
+    silent: true
+    vars:
+      GO_WORK: "go.work"
+      TMP_GO_WORK: "tmp_go.work"
+      LATEST_OPSLEVEL_GO_VERSION:
+        sh: go list -u -m -versions github.com/opslevel/opslevel-go/v2023 | awk '{print $NF}'
+    cmds:
+      - if [[ -f {{.GO_WORK}} ]]; then mv {{.GO_WORK}} {{.TMP_GO_WORK}}; fi
+      - defer: if [[ -f {{.TMP_GO_WORK}} ]]; then mv {{.TMP_GO_WORK}} {{.GO_WORK}}; fi
```

Comment:
For this task, `go.work` is briefly moved so `go list` finds the `.Version` in `go.mod` instead of a `.Path` to the submodule.